src/CompanyService.Broker/Consumers/GetPositionsConsumer.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using LT.DigitalOffice.CompanyService.Data.Interfaces;
+using LT.DigitalOffice.CompanyService.Models.Db;
+using LT.DigitalOffice.Kernel.Broker;
+using LT.DigitalOffice.Models.Broker.Models.Company;
+using LT.DigitalOffice.Models.Broker.Requests.Company;
+using LT.DigitalOffice.Models.Broker.Responses.Company;
+using MassTransit;
+
+namespace LT.DigitalOffice.CompanyService.Broker.Consumers
+{
+  public class GetPositionsConsumer : IConsumer<IGetPositionsRequest>
+  {
+    private readonly IPositionRepository _repository;
+
+    private object GetPosition(IGetPositionsRequest request)
+    {
+      List<DbPosition> positions = _repository.Get(request.PositionsIds, true);
```

Comment:
suggest _repository.Get(request.PositionsIds, includeUsers: true); to see what second arg means

Reply:
fixed